src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCosts = 0;
+  const carRentalPrice = 40;
+  const discount1 = 20;
+  const discount2 = 50;
```

Comment:
You should rename your variables. They have to describe what they are and why we need them
You may try sth like minDiscount and maxDiscount.